packages/block/test/block.spec.ts

```diff
@@ -5,6 +5,7 @@ import { Block, BlockBuffer, BlockHeader } from '../src'
 import blockFromRpc from '../src/from-rpc'
 import { Mockchain } from './mockchain'
 import { createBlock } from './util'
+import testnetMerge from './testdata/testnetMerge.json'
```

Comment:
oh nice i guess we don't need the `import * as` syntax as used for the other json files, would be nice to update the lines below so they all match :)

Reply:
I _think_ that these are two different things, `import * as foo from 'foo'` is when you import all modules with an alias (so then you can access modules with `foo.module1; foo.module2;` etc.) and `import foo from 'foo'` is when `foo` provides a default export

Reply:
Actually I take that back, it looks like both work, at least for JSON files!